examples/40_advanced/example_text_preprocessing.py

```diff
@@ -0,0 +1,76 @@
+# -*- encoding: utf-8 -*-
+"""
+==================
+Text Preprocessing
+==================
+This example shows, how to use text features in *auto-sklearn*. *auto-sklearn*
+can automatically encode text features if they are provided as string type.
+
+For processing text features you need a pandas dataframe and set the desired
+text columns to string and the categorical columns to category.
+
+*auto-sklearn* ass text embedding creates a bag of words count
+(https://scikit-learn.org/stable/tutorial/text_analytics/working_with_text_data.html).
+"""
+
+import matplotlib.pyplot as plt
```

Comment:
Could you please check whether all imports are needed? I think you don't need plt, right?